### ISSUE TITLE
Installation instructions should be simpler

### ISSUE THREAD

**Issue by julienrf** [coursier/coursier#2213]
The [installation instructions](https://get-coursier.io/docs/cli-installation) show the following complete procedure:

~~~ sh
$ curl -fLo cs https://git.io/coursier-cli-"$(uname | tr LD ld)"
$ chmod +x cs
$ ./cs install cs
$ rm cs
~~~

Would it be possible to cut them to something just like the following?

~~~ sh
$ curl -s "https://git.io/coursier-cli" | bash
~~~

**Comment by alexarchambault** [comment]
I agree it would be great to have such a script. The Scala CLI scripts [`scala-setup.sh`](https://github.com/VirtusLab/scala-cli-packages/blob/d6912816a8c721818ec4743e88a490563930390d/scala-setup.sh) and [`scala-cli.sh`](https://github.com/VirtusLab/scala-cli/blob/868e6b3dde1e3d246ac9fc374a766713b72d2e96/scala-cli.sh) can be used as a source of inspiration (the Scala CLI and coursier launchers are now packaged in very similar ways).

Note that the installation instructions [slightly changed recently](https://github.com/coursier/coursier/commit/e57f4e64ac047ee55c622f5aa694e5b235346239#diff-f01172f1b5ea67035923ec11821ea265f853f17de6143a07d1207e810fe9995a), this is already reflected [on the website](https://get-coursier.io/docs/cli-installation).

(Also, I had made [this](https://github.com/coursier/ci-scripts/blob/master/setup.sh) some time ago, installing `cs` in the current directory, but it's less handy, and it relies on non-compressed coursier launchers, which are not pushed any more, so it's deprecated too.)
